packages/manager/modules/web-paas/src/details/service/add-addon/routing.js

```diff
@@ -0,0 +1,41 @@
+import find from 'lodash/find';
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('web-paas.dashboard.service.add-addon', {
+    url: '/:projectId/:addonType',
```

Comment:
```suggestion
    url: '/:addonType',
```
`projectId` parameter should be on `web-paas.dashboard` state I think